python/tvm/micro/model_library_format.py

```diff
@@ -317,6 +321,16 @@ def _export_graph_model_library_format(
         "style": "full-model",
     }
 
+    if is_aot and (str(mod.runtime) == "crt"):
+        standalone_crt = {
+            "short_name": "tvm_standalone_crt",
+            "url": "./runtime",
```

Comment:
here ^

Reply:
Thanks for the feedback @leandron . I've changed this now.